tutorecommerce/templates/ecommerce/apps/tutor.py

```diff
@@ -61,6 +78,22 @@
 COMPRESS_OFFLINE = True
 
 PAYMENT_PROCESSOR_CONFIG = {
-    "openedx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
+    "edx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
 }
-PAYMENT_PROCESSORS = list(PAYMENT_PROCESSORS) + {{ ECOMMERCE_EXTRA_PAYMENT_PROCESSOR_CLASSES }}
\ No newline at end of file
+PAYMENT_PROCESSORS = list(PAYMENT_PROCESSORS) + {{ ECOMMERCE_EXTRA_PAYMENT_PROCESSOR_CLASSES }}
+
+SCAR_DEFAULT_CURRENCY = 'CNY'
```

Comment:
`SCAR_DEFAULT_CURRENCY` -> `OSCAR_DEFAULT_CURRENCY`

This should be a configurable setting that would be different for each user. Same for `CURRENCY_SYMBOL`.